blog/2025/09-30-okd-4.20-release-notes.md

```diff
@@ -0,0 +1,531 @@
+
+***
+
+## OpenShift Cloud Update Summary
+
+This release includes several enhancements related to Machine API (MAPI) to Cluster API (CAPI) migration, network policies, and AWS support.
+
+**MAPI to CAPI Migration Enhancements (Tech Preview Phase 2)**
+
+* **Automated Machine Deletion During Migration**: Support for **deleting machines during MAPI to CAPI migration** has been added, ensuring proper cleanup of associated resources.
+* **Defaulting Authoritative API**: The `status.authoritativeAPI` field for MAPI Machines/MachineSets is now **automatically populated** to prevent reconciliation conflicts.
+* **Deletion Synchronization**: **Deletion mechanics for MAPI and CAPI mirrored resources have been improved** using a finalizer for synchronization.
+* **Restricting Non-Authoritative Resource Updates**: Updates to non-authoritative MAPI Machines/MachineSets are now **restricted** to labels, annotations, and authoritative API related fields.
+
+**Network Policy Enhancements**
+
+* **Bespoke Network Policies for CAPI and MAPI**: **New network policies have been implemented** for CAPI and MAPI namespaces to enhance security.
+* **Network Policies for CCM Namespaces**: **Default deny network policies** have been added to Cloud Controller Manager (CCM) related namespaces.
+
+**AWS Enhancements (Tech Preview Phase 2)**
+
+* **Deprecated `DeleteOnTermination` for EBS Volumes**: The field for non-root EBS volumes in `AWSMachineSpec` has been deprecated.
+* **Restricted `AWSMachineSpec` Fields**: A Validating Admission Policy (VAP) now **prevents the use of several unsupported fields** in `AWSMachineSpec`.
+
+**Other Improvements (GA)**
+
+* **Must-gather Extended for CAPI Resources**: The `must-gather` tool now **collects relevant Cluster API resources** to aid in debugging machine-related issues.
+* **Prioritized Machine Deletion**: Machines annotated with `machine.openshift.io/delete-machine=true` are now **prioritized for deletion**.
+
+***
+
+## Network Edge Updates
+
+## OpenShift Service Mesh (OSSM) and Istio Updates
+
+* **OSSM and Istio Version Bumps**: OpenShift 4.20 now uses **OSSM 3.1.0 and Istio 1.26.2**.
+* **OSSM Override (Unsupported)**: An unsupported override annotation has been added to `GatewayClass` to allow specifying a custom OSSM channel and version, intended for testing and development.
+* **Conflicting OSSM Subscription Detection**: The Cluster Ingress Operator now **detects conflicting OSSM subscriptions** (e.g., OSSM 2.x) that prevent Gateway API deployment.
+
+## Gateway API Enhancements
+
+* **Gateway API Inference Extension (GIE) Support**: Gateway API Inference Extension (GIE) is now **enabled on OpenShift 4.20** when the `InferencePool` CRD is present, enabling integration with AI workloads.
+
+***
+
+## Multiple Architecture Enablement Updates
+
+This release focuses on improving imagestream behavior in multi-architecture clusters.
```

Comment:
hmm..this is still behind TechPreviewNoUpgrade gate..this has been wrongly classified as a GA feature